### ISSUE TITLE
Is there a way to get the actual probability of  each class from the model prediction?

### ISSUE THREAD

**Issue by brunoedcf** [thunlp/OpenPrompt#256]
from openprompt import PromptForClassification
promptModel = PromptForClassification(
    template = promptTemplate,
    plm = plm,
    verbalizer = promptVerbalizer,
)

import torch

promptModel.eval()
with torch.no_grad():
     for batch in data_loader:
         logits = promptModel(batch)
         preds = torch.argmax(logits, dim = -1)
         print(tokenizer.decode(batch['input_ids'][0], skip_special_tokens=True), classes[preds])

I have 3 classes and I want the probability of each one instead of the the higher one.

**Comment by Protiva** [comment]
The logits variable gives you the probability of each class.  

preds = torch.argmax(logits, dim = -1) is selecting the class with the highest value.

**Comment by xiyang-aads-lilly** [comment]
@brunoedcf did you figure out how to?  I think we can apply softmax on logits to get probability distribution.  Any thoughts?
